ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -282,11 +289,44 @@ const AdminUser = withUrlParams()(class extends React.Component<Props, State> {
     });
   }
 
+  // returns the updated profile value only if it has changed
+  updatedProfileValue(attribute: string) {
+    const oldValue = fp.get(['oldProfile' , attribute], this.state);
+    const updatedValue = fp.get(['updatedProfile' , attribute], this.state);
+    if (!fp.isEqual(oldValue, updatedValue)) {
+      return updatedValue;
+    }
+  }
+
+  updateAccountProperties() {
+    const {updatedProfile} = this.state;
+    const {username} = updatedProfile;
+    const request: AccountPropertyUpdate = {
+      username,
```

Comment:
`username` and `accessBypassRequests` are required.  Only include others if they have changed (and are implemented in this PR)